src/bin/juliainstaller.rs

```diff
@@ -234,17 +256,23 @@ pub fn main() -> Result<()> {
     }
 
     let mut install_choices = InstallChoices {
-        backgroundselfupdate: 0,
-        startupselfupdate: 1440,
+        backgroundselfupdate: args.background_selfupdate_interval.parse().unwrap(),
+        startupselfupdate: args.startup_selfupdate_interval.parse().unwrap(),
         symlinks: false,
-        modifypath: true,
+        modifypath: args.add_to_path.eq("yes"),
         install_location: dirs::home_dir()
             .ok_or(anyhow!("Could not determine the path of the user home directory."))?
             .join(".juliaup"),
         modifypath_files: find_shell_scripts_to_be_modified(true)
             .with_context(|| "Failed to identify the shell scripts that need to be modified.")?,
     };
 
+    // args.alternate_path is used to overwrite the default install location (if
+    // specified). This is also the location use if disable_confirmation_prompt
+    if ! args.alternate_path.is_empty() {
```

Comment:
I think instead of checking for an empty string, there might be a way to use an option and check that?

Reply:
Makes sense now that I've learned quite a bit more about rust. Using `Option<String>` + a `match` statement now.